tests/py/dynamo/models/test_dyn_models.py

```diff
@@ -66,6 +67,7 @@ def forward(self, x):
         torch.cuda.empty_cache()
 
 
+@unittest.skip("Skipping this test for now due to a bug")
```

Comment:
Can you add the reason for the failure to this message?

Reply:
I opened an issue for the bug: https://github.com/pytorch/TensorRT/issues/2794